tasks/setup_RedHat.yml

```diff
@@ -19,6 +20,7 @@
     dest: "{{ apache_config_path }}/conf.d/00-envvars.conf"
     line: 'Define APACHE_LOG_DIR /var/log/httpd'
     create: yes
+    mode: 0755
```

Comment:
Do we need to make this file executable or is `0644` sufficient?

Reply:
Fixed, i was misled by the "DIR" in the task name and clicked the wrong button on the automation keyboard :)